### DISCUSSION TITLE
not overriding visibility layers of camera 3d

### DISCUSSION THREAD

**Discussion by KayEntry17** [ramokz/phantom-camera#362] [Question / Help Wanted]
my project relies on using visual layers for a mechanic based on switching dimensions, is there any way not to use camera 3d resource? if I leave the field empty it just activates all layers

**Comment by ramokz** [comment]
You should be able to achieve that result by removing the default value in the property declaration in `phantom_camera_3d.gd` on line `267`, so it looks like:
```gd
@export var camera_3d_resource: Camera3DResource:
  set = set_camera_3d_resource,
  get = get_camera_3d_resource
```

It should probably be like that by default, just to make the choice of using this more explicit, so can look into changing that for the upcoming release.
